### DISCUSSION TITLE
Astroid installed/create a “template_name” folder inside the main Joomla root/images folder?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#278] [General]
I remark that Astroid installed/create a **“template_name” folder** with some images inside the main **Joomla Root/images** folder.

I hate components that clutter the **Joomla Root/images** folder because it confuse websites authors and editors. To me, nothing other than Joomla functionalities folders and folders reserve to website editors should be present.

**Can we remove them because anyway the **same images are present** in the **Joomla Root/media** Astroid template folder?**

![Screen Shot 2023-01-15 at 5 16 09 PM](https://user-images.githubusercontent.com/1907276/212570242-54440bdb-f993-4025-8cb5-82e28a5a1521.jpg)

**Comment by deltapapa01** [comment]
Hello,
I don't think so at all, I think it's very good and also important that we have the option of storing template files in the image folder. I can edit and adjust them conveniently from the backend without having to set any additional rights or install additional programs. This is extremely important, especially when you're on the go, don't have FTP access and just have to make a css adjustment. This is also the reason why all major template frameworks allow this option.

But if it bothers you, why not just delete the Astroid template folder in the images folder? And then you refer to the setting on the other media folder.

I think it's great that you have all the options, and I wouldn't like it if the Astroid Template folder no longer appears under the images folder.

WBR,
deltapapa

**Reply by Chacapamac** [reply]
Thanks, @deltapapa01 , 
Yes, I agree with You, this is good points, more flexibility is always better.

If you have time, can you precise how you use that feature, I’m not too sure to see what “files, you want to put there?

I already trash the folders.
